drivers/interrupt_controller/Kconfig.nxp_irqsteer

```diff
@@ -11,3 +11,8 @@ config NXP_IRQSTEER
 		multiple interrupts from peripheral to one or
 		more CPU interrupt lines. This is used for CPUs
 		such as XTENSA DSPs.
+
+config USE_NXP_MCUX_SDK_IRQSTEER1
+	bool "Use IRQSTEER1 driver in NXP MCUX SDK"
+	default n
+	depends on NXP_IRQSTEER
```

Comment:
is this a different version of this IRQSTEER or a completely different IP? because for example with EDMA we use different drivers but put a version property in DT to pick which one, take a look at EDMA kconfigs file for example

Reply:
> is this a different version of this IRQSTEER or a completely different IP? because for example with EDMA we use different drivers but put a version property in DT to pick which one, take a look at EDMA kconfigs file for example

Fixed it after following the rules from mcux edma driver.